aspnetcore/signalr/javascript-client.md

```diff
@@ -0,0 +1,102 @@
+---
+title: ASP.NET Core SignalR JavaScript client
+author: rachelappel
+description: Overview of ASP.NET Core SignalR JavaScript client.
+manager: wpickett
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 04/06/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/javascript-client
+---
+
+# ASP.NET Core SignalR JavaScript client
+
+By [Rachel Appel](http://twitter.com/rachelappel)
+
+The ASP.NET Core SignalR JavaScript client library enables developers to call server-side hub code.
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/live/aspnetcore/signalr/javascript-client/sample) ([how to download](xref:tutorials/index#how-to-download-a-sample))
+
+## Install the SignalR client package
+
+The SignalR JavaScript client library is delivered as an [npm](https://www.npmjs.com/) package. If you're using Visual Studio, run `npm install` from the command line in any folder. For Visual Studio Code, run the command from the **Integrated Terminal**.
```

Comment:
- Since most Visual Studio users don't like leaving the IDE, have them use the Package Manager Console window.
- The *package.json* file needs to exist before `npm install` will succeed. Add the following command before that one: `npm init -y`.